src/prefect/infrastructure/docker.py

```diff
@@ -98,6 +83,39 @@ def _get_client(self):
         return docker_client
 
 
+class DockerRegistry(BaseDockerLogin):
+    """
+    Connects to a Docker registry.
+
+    Requires a Docker Engine to be connectable. Login information is persisted to disk
+    at the Docker default location.
+
+    Attributes:
+        username: The username to log into the registry with.
+        password: The password to log into the registry with.
+        registry_url: The URL to the registry. Generally, "http" or "https" can be
+            omitted.
+        reauth: If already logged into the registry, should login be performed again?
+            This setting defaults to `True` to support common token authentication
+            patterns such as ECR.
+    """
+
```

Comment:
Human friendly block type name to remain consistent with what was already merged. It's nice that the `_logo_url` is on `BaseDockerLogin` since it gives a default image to any subclasses.
```suggestion
_block_type_name = "Docker Registry"
```

Reply:
Oops, thanks!